packages/aws-rfdk/lib/deadline/lib/worker-configuration.ts

```diff
@@ -86,6 +106,8 @@ export interface WorkerInstanceConfigurationProps {
    * @default The Worker is assigned the default settings as outlined in the WorkerSettings interface.
    */
   readonly workerSettings?: WorkerSettings;
+
+  readonly userDataProvider?: IInstanceUserDataProvider;
```

Comment:
I'm surprised this got through the linting. This needs a docstring.

Reply:
I missed it but surprised as well that linting missed it too.